src/test/rustdoc/type-layout.rs

```diff
@@ -52,3 +52,10 @@ pub struct Unsized([u8]);
 
 // @!has type_layout/trait.MyTrait.html 'Size: '
 pub trait MyTrait {}
+
+// @has type_layout/enum.Variants.html '<code>A</code>: 0 bytes'
```

Comment:
Can you add another `@has` check that ensures the size of the enum as a whole is shown?